test/Riok.Mapperly.Tests/TestSourceBuilderOptions.cs

```diff
@@ -26,6 +27,9 @@ public record TestSourceBuilderOptions(
     public static TestSourceBuilderOptions WithIgnoreObsolete(IgnoreObsoleteMembersStrategy ignoreObsoleteStrategy) =>
         new(IgnoreObsoleteMembersStrategy: ignoreObsoleteStrategy);
 
+    public static TestSourceBuilderOptions WithMemberList(RequiredMapping requiredMappingStrategy) =>
```

Comment:
I'd use the same name for this method as for the functionality itself `WithRequiredMapping`